packages/remix-server-runtime/server.ts

```diff
@@ -119,12 +118,12 @@ export const createRequestHandler: CreateRequestHandlerFunction = (
 
       response = await handleDataRequestRR(
         serverMode,
+        _build,
```

Comment:
Pass the full `build` through - this aligns slightly closer to the signature for `handleDocumentRequest`